sync/class.jetpack-sync-module-plugins.php

```diff
@@ -30,6 +31,16 @@ public function init_before_send() {
 	}
 
 	public function check_upgrader( $upgrader, $details) {
+		if (
+			isset( $details['type'] ) &&
+			'plugin' == $details['type'] &&
+			! empty ( $upgrader->skin->get_errors() )
```

Comment:
We can't assume that every skin has get_errors method. Plugins could have custom skins classes implemented. For example. I think we should also check skin->result for errors.